### ISSUE TITLE
Doesn't seem to handle floats at all

### ISSUE THREAD

**Issue by kke** [jjtara/Library-Arduino-Cbor#8]
A lot of sensor data comes as floats and the spec seems to include several flavors of floats.

**Comment by jspark311** [comment]
No. It doesn't. I hard-forked the upstream into my own project to handle float and double.

https://github.com/Manuvr/ManuvrOS/tree/master/ManuvrOS/Types/cbor-cpp

If the maintainer will take a PR, I will back-port my changes in a manner compatible with the rest of the library.
